2024/src/yaml/data.yaml

```
@@ -20,10 +20,10 @@ about:
     #   end_at: ??:??
     #   session_types:
     #     - Workshop
-    # - begin_at: ??:??
-    #   end_at: ??:??
-    #   session_types:
-    #     - Presentation
+        - begin_at: 13:30
+          end_at: 18:00
+          session_types:
+            - Presentation
```

インデントを 2 段階上げて、 `parts[0].session_types[]` ではなく `parts[0]` となるようにしてください

```suggestion
    - begin_at: 13:30
      end_at: 18:00
      session_types:
        - Presentation
```